### ISSUE TITLE
Use of the element b instead of em (WCAG 1.3.1d)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1223]
"Speaker notes" is marked with the element `b`. `Strong` or `em` should be used instead of `b`.

https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit

**Comment by aoelen** [comment]
Fixed in #1248
